extensions/kubernetes/spi/src/main/java/io/quarkus/kubernetes/spi/KubernetesCommandBuildItem.java

```diff
@@ -1,22 +1,30 @@
 package io.quarkus.kubernetes.spi;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
 import io.quarkus.builder.item.SimpleBuildItem;
 
 public final class KubernetesCommandBuildItem extends SimpleBuildItem {
 
-    private final String command;
-    private final String[] args;
+    private final List<String> command;
+    private List<String> args = Collections.emptyList();
```

Comment:
Please don't make it non-final. Instead use the constructor, static factory methods or introduce a builder

Reply:
Changed. 
I've created a few factory methods.
Let me know what you think now.